papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_1.yaml

```diff
@@ -41,9 +41,9 @@ spec:
             python setup.py develop;
             cd ../zdm;
             git fetch;
-            git checkout H0_paper;
+            git checkout full_craco;
```

Comment:
If this is being merged back to H0_paper, better make sure you checkout H0_paper!